### PR TITLE
Feat: Add setting for using lowercase v.s. uppercase letters

### PR DESCRIPTION
**Pull request by danielgolden**
#### TLDR
Wanted to be able to play in uppercase as well as lowercase mode so this PR adds that small setting.

### Description
Hi, Steve ✌🏽. I played with my 4 and 3 year olds this morning who I'v been trying to teach the alphabet. Unfortunately, I haven't yet taught my 4 year old lowercase letters yet so she had a hard time. I thought it would be cool to be able to play in uppercase mode as an alternative. So here's what's included in this PR:
- Adds a small "settings" button in the bottom right corner
- Adds a <Settings  /> component to handle the new option

### Preview
https://user-images.githubusercontent.com/812989/178157586-d99b36a9-24aa-4311-804f-13e3eb22bd5b.mp4

---
**Note:** I fully understand if this isn't the direction you want to take the project. Thanks for making a fun thing :). My kids and I are enjoying it!

### CODE REVIEW

**Comment by steveruizok** [comment]
No this works! I'd been wondering how to display settings without too much UI interference, I think this works. Thanks for the PR!
